app/webpacker/components/RegistrationsV2/Register/CompetingStep.jsx

```diff
@@ -397,8 +400,8 @@ export default function CompetingStep({
                   setGuests(Number.parseInt(data.value, 10));
                 }}
                 min="0"
-                max={guestsRestricted && (competitionInfo.guests_per_registration_limit ?? 99)}
-                error={guestsRestricted && Number.isInteger(competitionInfo.guests_per_registration_limit) && guests > competitionInfo.guests_per_registration_limit && I18n.t('competitions.competition_info.guest_limit', { count: competitionInfo.guests_per_registration_limit })}
+                max={guestLimit}
+                error={guestsRestricted && guests > guestLimit && I18n.t('competitions.competition_info.guest_limit', { count: guestLimit })}
```

Comment:
Don't we want to show a (different) error message when guests are not restricted? Might as well handle it in the front end since we have the limit (and are using it as `max`).

Reply:
Gregor and I decided to increase the limit to 99, for the purposes of getting this (potentially production-breaking) problem fixed - there is definitely more we can do here to improve QoL for organizers and competitors - I've updated #10543 with a low-priority item that covers thinking more deeply about handling this well